torch/_torch_docs.py

```diff
@@ -2339,6 +2339,30 @@ def merge_dicts(*dicts):
     tensor(7)
 """)
 
+add_docstr(torch.vdot,
+           r"""
+vdot(input, other) -> Tensor
+
+Computes the dot product (inner product) of two tensors.
+
+.. note:: This function does not :ref:`broadcast <broadcasting-semantics>`.
+
+.. note:: The vdot(a, b) function handles complex numbers differently than dot(a, b).
+          If the arguments are complex, the complex conjugate of the first argument
+          is used for the calculation of the dot product.
+
+Example::
+
+    >>> torch.vdot(torch.tensor([2, 3]), torch.tensor([2, 1]))
+    tensor(7)
+    >>> a = torch.tensor((1 +2j, 3 - 1j))
+    >>> b = torch.tensor((2 +1j, 4 - 0j))
+    >>> torch.vdot(a, b)
+    tensor([16.+1.j])
+    >>> torch.vdot(b, a)
+    tensor([16.-1.j])
+""")
```

Comment:
If you do use a construction like {out} then you'll need to add a `.format(` here.